src/assets/alignmentD3Tests.js

```diff
@@ -314,17 +353,29 @@ var getMisalignmentCountCaller = function(
  * @param {Array}  collection - HTMLCollection of features
```

Comment:
What does an HTMLCollection of features look like? Since this is the main function called from the tests, it should be well documented what the parameters look like. A code example of how you would get such a collection would be helpful. For example, in the heat map tests I think the code is `const cellsCollection = document.querySelectorAll('.cell');`? That code sample, along with an explanation that it's getting all the D3 data points, would be helpful to understand what the param is supposed to be.